client/core/wallet.go

```diff
@@ -152,6 +154,7 @@ func (w *xcWallet) state() *WalletState {
 		Address:      w.address,
 		Units:        winfo.UnitInfo.AtomicUnit,
 		Encrypted:    len(w.encPass) > 0,
+		PeerCount:    w.peerCount,
```

Comment:
Also to do if this seems good is to show a specific icon for when peerCount == 0, perhaps just the existing red "!" icon with appropriate tooltip.